package/src/languageService/kustoLanguageService.ts

```diff
@@ -1068,6 +1067,13 @@ class KustoLanguageService implements LanguageService {
             KustoLanguageService.toBridgeList([...scalarSymbols, ...tabularSymbols])
         );
         this._script = this._script?.WithGlobals(this._kustoJsSchemaV2);
+
+        // Set parameters is only working with the below code. It didn't used to need this, why does it now?!?
+        // Copy+pasted from setSchema
+        const kustoJsSchema = KustoLanguageService.convertToKustoJsSchema(this._schema);
```

Comment:
Are you sure this is needed? 
did you try only calling to createRulesProvider with existing kusto schema? 
this.createRulesProvider(this._kustoJsSchema, this._schema.clusterType);